.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -8,18 +17,17 @@ Need Support?
 * Have a feature request? Please post it on [User Voice](https://feedback.azure.com/forums/907045-azure-iot-edge) to help us prioritize.
 * Have a technical question? Ask on [Stack Overflow](https://stackoverflow.com/questions/tagged/azure-iot-edge) with tag "azure-iot-edge".
 * Need Support? Every customer with an active Azure subscription has access to [support](https://docs.microsoft.com/en-us/azure/azure-supportability/how-to-create-azure-support-request) with guaranteed response time.  Consider submitting a ticket and get assistance from the Microsoft support team.
```

Comment:
```suggestion
* Need Support? Every customer with an active Azure subscription has access to [support](https://docs.microsoft.com/en-us/azure/azure-supportability/how-to-create-azure-support-request) with guaranteed response time.  Consider submitting a ticket and get assistance from the Microsoft support team.
```
* Need support? Azure customers with support plans have access to priority technical support directly from the Azure Portal. Support plans details are at: https://azure.microsoft.com/en-us/support/plans/